fabric-extensibility-api-v1/src/main/java/net/fabricmc/fabric/mixin/extensibility/AbstractSkeletonEntityMixin.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.mixin.extensibility;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Redirect;
+
+import net.minecraft.entity.mob.AbstractSkeletonEntity;
+import net.minecraft.item.BowItem;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.Items;
+
+@Mixin(AbstractSkeletonEntity.class)
+public class AbstractSkeletonEntityMixin {
+	@Redirect(method = "updateAttackType", at = @At(value = "INVOKE", target = "Lnet/minecraft/item/ItemStack;getItem()Lnet/minecraft/item/Item;"))
```

Comment:
~~So I'm a little wary of this mixin because if I'm understanding it correctly this mixin causes skeletons to adopt whatever attack method corresponds to their held item instead of only being able to use bows, and while I could definitely see a context where this would be helpful, for me the goal of fapi is to provide helpful hooks, not write features into the game, and this would be a departure from vanilla functionality which is why I'm not sure it would quite fit inside of fapi.~~

Ignore all of that I figured out that this just allows skeletons to use custom bow items.